evmd/cmd/evmd/config/config.go

```diff
@@ -1,36 +1,41 @@
 package config
 
 import (
+	"github.com/cosmos/evm/server/config"
 	"github.com/cosmos/evm/types"
 	evmtypes "github.com/cosmos/evm/x/vm/types"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// ChainsCoinInfo is a map of the chain id and its corresponding EvmCoinInfo
-// that allows initializing the app with different coin info based on the
-// chain id
-var ChainsCoinInfo = map[uint64]evmtypes.EvmCoinInfo{
-	EighteenDecimalsChainID: {
-		Denom:         ExampleChainDenom,
-		ExtendedDenom: ExampleChainDenom,
-		DisplayDenom:  ExampleDisplayDenom,
-		Decimals:      evmtypes.EighteenDecimals,
-	},
-	// SixDecimalsChainID provides a chain ID which is being set up with 6 decimals
-	SixDecimalsChainID: {
-		Denom:         "utest",
-		ExtendedDenom: "atest",
-		DisplayDenom:  "test",
-		Decimals:      evmtypes.SixDecimals,
-	},
-	// EVMChainID provides a chain ID used for internal testing
-	EVMChainID: {
-		Denom:         "atest",
-		ExtendedDenom: "atest",
-		DisplayDenom:  "test",
-		Decimals:      evmtypes.EighteenDecimals,
-	},
+// GetEvmCoinInfo returns appropriate EvmCoinInfo for evmd based on chainID.
+func GetEvmCoinInfo(chainID uint64) evmtypes.EvmCoinInfo {
```

Comment:
Could we just export something like this once from the library instead of here and how it is duplicated in `testutil`?